README.md

```diff
@@ -1,81 +1,119 @@
+For [pyright](CONFIG.md#pyright), this is any directory containing one of:
+
+- `.git`,
+- `setup.py`,
+- `setup.cfg`,
+- `pyproject.toml`, or
+- `requirements.txt`.
 
-Language servers require each project to have a `root` in order to provide completion and search across symbols that may not be defined in your current file, and to avoid having to index your entire filesystem on each startup.
+Language servers require each project to have a `root` in order to provide completion and search across symbols that may not be defined in your current file; it also avoids having to index your entire filesystem on each startup.
 
 ## Enabling additional language servers
 
-Enabling most language servers is as easy as installing the language server, ensuring it is on your PATH, and adding the following to your config:
+Most language servers can be installed in less than a minute.
+Enabling a language server means:
+
+1. Installing the language server binary: `npm install global pyright`;
+1. Ensuring the language server can be found on your `PATH`: typing `pyright --version` succeeds;
+1. Adding the language server configuration to your `init.vim` file:
 
 ```vim
 lua << EOF
-require'lspconfig'.rust_analyzer.setup{}
+require'lspconfig'.pyright.setup{}
 EOF
 ```
-For a full list of servers, see [CONFIG.md](CONFIG.md). This document contains installation instructions and additional, optional customization suggestions for each language server. For some servers that are not on your system path (jdtls, elixirls) you will be required to manually add `cmd` as an entry in the table passed to setup. Most language servers can be installed in less than a minute.
+
+For a full list of servers, see [CONFIG.md](CONFIG.md).
+This document contains installation instructions and optional customizations for each language server.
+For some servers that are not on your system path, e.g., `jdtls`, `elixirls`, you will be required to manually add `cmd` as an entry in the table passed to setup.
 
 ## Keybindings and completion
 
-nvim-lspconfig does not map keybindings or enable completion by default. Manual, triggered completion can be provided by neovim's built-in omnifunc. For autocompletion, a general purpose [autocompletion plugin](https://github.com/neovim/nvim-lspconfig/wiki/Autocompletion) is required. The following example configuration provides suggested keymaps for the most commonly used language server functions, and manually triggered completion with omnifunc (\<c-x\>\<c-o\>). 
+nvim-lspconfig does not map keybindings or enable completion by default.
+Manually-triggered completion can be provided by Neovim's built-in [https://neovim.io/doc/user/options.html#'omnifunc'](omnifunc).
+For autocompletion, a general purpose [autocompletion plugin](https://github.com/neovim/nvim-lspconfig/wiki/Autocompletion) is required.
 
-```vim
+Copy the following example configuration into your `init.vim` file.
```

Comment:
Seems a little imperative, the example config isn't required by any means, it's just a suggestion

Reply:
I wanted to make sure to init.vim to highlight that this is for .vim files only, init.lua this would not work. Either that, or we include init.lua examples as well for each?

Reply:
init.vim/init.lua examples are identical with the exception of the lua << EOF/ EOF pairing